### PR TITLE
Add threshold to tracker trend and AMS

### PR DESCRIPTION
**Pull request by mgrauer**

### CODE REVIEW

**Comment by mgrauer** [comment]
Ready for QA.

File: modules/tracker/models/base/AggregateMetricSpecModelBase.php [hunk 1]
```diff
@@ -35,8 +35,10 @@ public function __construct()
             'name' => array('type' => MIDAS_DATA),
             'description' => array('type' => MIDAS_DATA),
             'spec' => array('type' => MIDAS_DATA),
-            'value' => array('type' => MIDAS_DATA),
-            'comparison' => array('type' => MIDAS_DATA),
```
Comment (cpatrick) [inline modules/tracker/models/base/AggregateMetricSpecModelBase.php]: This confused me for a bit, but I can see it's vestigial cleanup from the 2.0.1 upgrade. I'm saying that here to confirm my understanding is correct.

**Comment by cpatrick** [comment]
This LGTM. Is the plan to add API/UI later?

File: modules/tracker/models/base/AggregateMetricSpecModelBase.php (hunk 1 above)
Comment (mgrauer) [inline modules/tracker/models/base/AggregateMetricSpecModelBase.php]: exactly right

**Comment by mgrauer** [comment]
Yes, let's discuss API and UI at the demo.

**Comment by mgrauer** [comment]
The plan for API/UI is to roll this out, then see how it fits in with upcoming changes to Key Metrics.

An API would be relatively easy to add, but the UI could take some time and needs to be thought through more.

**Pull request merged**